src/js/view/controls/controls.js

```diff
@@ -173,7 +173,10 @@ export default class Controls extends Events {
         // Controlbar
         const controlbar = this.controlbar = new Controlbar(api, model,
             this.playerContainer.querySelector('.jw-hidden-accessibility'));
-        controlbar.on(USER_ACTION, () => this.userActive());
+        controlbar.on(USER_ACTION, () => {
+            this.once('userInactive', () => this.focusPlayerElement(), this);
```

Comment:
you don't need the arrow here. With `this` context: `this.once('userInactive', this.focusPlayerElement, this);` will do.  Then you can also call `off` in exactly the same way.

Reply:
You might want to also call `off` before registering `once` in case it is possible for multiple `USER_ACTION` events with fewer `blurCallback`s.